docs/desktop-wpf/xaml-services/basic-reading-writing.md

```diff
@@ -8,7 +8,7 @@ ms.assetid: 6ac27fad-3687-4d7a-add1-3e90675fdfde
 ---
 # XAMLServices class and basic XAML reading or writing
 
-<xref:System.Xaml.XamlServices> is a class provided by .NET Framework XAML Services that can be used to address XAML scenarios that don't require specific access to the XAML node stream or to XAML type system information obtained from those nodes. <xref:System.Xaml.XamlServices> API can be summarized as follows: `Load` or `Parse` to support a XAML load path, `Save` to support a XAML save path, and `Transform` to provide a technique that joins a load path and save path. `Transform` can be used to change from one XAML schema to another. This topic summarizes each of these API classifications and describes the differences between particular method overloads.  
+<xref:System.Xaml.XamlServices> is a class provided by .NET that can be used to address XAML scenarios that don't require specific access to the XAML node stream or to XAML type system information obtained from those nodes. <xref:System.Xaml.XamlServices> API can be summarized as follows: `Load` or `Parse` to support a XAML load path, `Save` to support a XAML save path, and `Transform` to provide a technique that joins a load path and save path. `Transform` can be used to change from one XAML schema to another. This topic summarizes each of these API classifications and describes the differences between particular method overloads.  
```

Comment:
Doesn't that imply that it's provided by all the .NET ecyosystem, which includes .NET Standard and Xamarin ?

The API docs <https://docs.microsoft.com/en-us/dotnet/api/system.xaml.xamlservices?view=netframework-4.8> state it's applied to .NET Core and .NET Framework, but doesn't mention .NET Standard or Xamarin

Reply:
It could be interpreted that way, but I don't think it needs to be. Saying something is from .NET doesn't necessarily mean it's supported by everything that implements .NET in some form. You'll still need to check any API for compatibility. .NET Standard is a specification set of APIs, by saying something is .NET compatible, you don't automatically imply it's supported by .NET Standard.

@mairaw What do you think?